nx-X11/programs/Xserver/os/utils.c

```diff
@@ -411,7 +411,9 @@ LockServer(void)
   if (lfd < 0)
     FatalError("Could not create lock file in %s\n", tmp);
   (void) sprintf(pid_str, "%10ld\n", (long)getpid());
-  (void) write(lfd, pid_str, 11);
```

Comment:
> Hardcoded magic number. Seems the right value and unlikely to change, though.

pid_str is defined as char[12], so using 11 here seems appropriate. Also backported from X.org Xserver.